.github/workflows/PR.yml

```diff
@@ -0,0 +1,20 @@
+name: Label and Assign PRs
+on:
+  pull_request:
+    types:
+      - opened
+jobs:
+  label_and_assign_PRs:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - run: |
```

Comment:
Try to echo some error message for failed runs of this workflow.

Reply:
Applies to the gh pr edit lines below.

Reply:
If you go under the Actions tab of the repo and click on the workflow run, it provides a log of the runs and any errors that occur already display there for any point of the run.

I could add 
```
- name: Check for errors
        if: ${{ failure() }}
        run: echo "::error::Step 'Label and Assign PRs' failed"
```

and change the message to be a custom message but that may be redundant

Reply:
I'm assuming that echo is logged within the log of the run. In which case, yes, that would be redundant, feel free to disregard and resolve.

Reply:
Also, we should probably have unit tests that validate this workflow, if possible.

Reply:
I agree we should implement unit tests

Reply:
I can see if that is a viable thing to do

Reply:
https://stackoverflow.com/questions/67707244/how-to-test-github-workflow-without-merging-into-master-main-branch

@OwenAEdwards